keycloak/kustomization.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+commonLabels:
+    app.kubernetes.io/name: keycloak
+resources:
+  - deployment.yaml
+  - service.yaml
+  - service-account.yaml
+  - ingress.yaml
+configMapGenerator:
+  - name: keycloak-config
+    literals:
+      - KC_HOSTNAME_URL=https://keycloak.hashbang.sh
+      - KC_DB=postgres
+      - KC_DB_URL_HOST=userdb-attempt-too-do-user-989073-0.db.ondigitalocean.com
+      - KC_DB_URL_PORT=25060
+      - KC_DB_URL_DATABASE=keycloak
+      - KC_DB_URL_PROPERTIES=?sslmode=require
```

Comment:
huh. this `?` is odd and barely undocumented

Reply:
It appends without the `?` unless it's provided